### ISSUE TITLE
Yosemite 10.10 + iPhoto 9.6

### ISSUE THREAD

**Issue by buklrr** [jawj/iphoto-flickr#16]
Hi guys,

When trying to upload my iPhoto Library using your script, I got the following error (even when doing sudo).

> sudo ./flickrbackup.rb 
> Authenticated as: buklrr1
> 102:133: execution error: iPhoto got an error: AppleEvent timed out. (-1712)
> 
> 0 photos in iPhoto library
> 0 photos not yet uploaded to Flickr
> 
> 287:292: syntax error: Expected “then”, etc. but found identifier. (-2741)
> 
> 0 standard albums in iPhoto

Was wondering if the script is ok with Mac OS Yosemite (10.10) + iPhoto 9.6 (910.29)...?

Thanks

**Comment by jawj** [comment]
I've added 10-minute timeouts to the two AppleScript blocks, so please update your script and try again, and tell me if that helps.

**Comment by edsmiley** [comment]
Still get a timeout.  I assume I can just increase that and see if that helps?  Thanks for putting it in the right spot!

**Comment by jawj** [comment]
Yes ... but with a pause that long I suspect there's some other underlying issue. Have you checked Console.app for potentially relevant messages while the script runs?

**Comment by edsmiley** [comment]
Actually, I changed the pause to 6000 seconds and it loaded up after about 14 minutes. Chugging along great now. I have tried 2 other pay solutions with no success. This looks like it is going to work!  Thanks a bunch.
